packages/api-generator/src/locale/en/v-navigation-drawer.json

```diff
@@ -16,13 +16,13 @@
     "temporary": "A temporary drawer sits above its application and uses a scrim (overlay) to darken the background",
     "touchless": "Disable mobile touch functionality",
     "value": "Controls whether the component is visible or hidden.",
-    "color": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "modelValue": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
+    "color": "Applies specified color to the control - it can be the name of material color (for example success or purple) or css color (#033 or rgba(255, 0, 0, 0.5)). You can find a list of built-in classes on the ([colors page.](https://next.vuetifyjs.com/en/styles/colors/#material-colors))",
```

Comment:
The team is working on a way to reuse props that are not part of a composable without duplicating this sentence for every component. 

Long story short, please revert this change.